systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -21,6 +21,16 @@
 
 ## Linked Site Page Changes
 
+Responsible for notifying a repository that a page that is linked to by an entity has changed.
```

Comment:
We should perhaps sync this line of description up with that which is currently in the table above.
I wonder if it should also only be in one of the 2 places.